worker/lib/workers/qemu.ts

```diff
@@ -257,6 +268,7 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 					'ip', ['addr', 'add', `${bridgeAddress}/24`, 'dev', bridgeName]
 				).on('exit', (code) => {
 					if (code == 0) {
+						console.log(`added ${bridgeAddress}/24 to ${bridgeName}...`);
```

Comment:
Also a debug statement (additionally, some were already committed that could probably be removed)